likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/universalfeed/view/LMFeedUniversalFeedFragment.kt

```diff
@@ -468,6 +463,8 @@ open class LMFeedUniversalFeedFragment : Fragment(), LMFeedUniversalFeedAdapterL
     }
 
     protected open fun onSearchIconClick() {
+        //todo: change this
```

Comment:
why it is on search icon clicked

Reply:
it was for testing only, this is fixed and moved to proper function in further PRs